package.json

```diff
@@ -25,6 +25,7 @@
     "lint": "xo src/**/*.js test/**/*.js",
```

Comment:
We should clean up scripts section. It contains not used actions like `build:docs`